iTunesUDataPurge/RemoveITunesUCourses.java

```diff
@@ -478,5 +501,186 @@ public static void fileUrl(String fAddress, String localFileName, String destina
 			}
 		}
 	}
+
+	public static void deletePage(String searchServer, String siteId, String username, String password) throws Exception{
+
+		String pageToDelete = "iTunes U";
+
+		// Create SOAP Connection
+		SOAPConnectionFactory soapConnectionFactory = SOAPConnectionFactory.newInstance();
+		SOAPConnection soapConnection = soapConnectionFactory.createConnection();
+
+		// Send SOAP Message to SOAP Server
+		String sakaiLoginUrl = searchServer + "sakai-axis/SakaiLogin.jws?wsdl";
+		String sakaiScriptUrl = searchServer + "sakai-axis/SakaiScript.jws?wsdl";
+
+		//Making the createLoginRequest call. Returns a response message containing the session id that should be active on server.
+		SOAPMessage soapResponse = soapConnection.call(createLoginRequest(searchServer, username, password), sakaiLoginUrl);
+
+		// print SOAP Response
+		System.out.print("Response SOAP Message:");
+		soapResponse.writeTo(System.out);
+
+		//Parses Soap Response to get session id as string
+		String sessionId = getSessionId(soapResponse);
+		System.out.println("");
+		System.out.println("Session Id:" + sessionId);
+
+		//Test validity of session id by calling Check Session function
+		soapResponse = soapConnection.call(createCheckSessionRequest(searchServer, sessionId), sakaiScriptUrl);
```

Comment:
Is it recommended that alway check the session validity after the createLoginRequest() call?